source/includes/_01-your-first-backend-in-10-minutes.md

```diff
@@ -0,0 +1,204 @@
+
+Similar to the `CreatePost` command we defined at the beginning, we will also need to define a few things:
+- Who can read the `Posts`
+- How the data is manipulated before returning it
+
+To make it easy, we will allow anyone to read it:
+```typescript
+@ReadModel({
+  authorize: 'all'// Specify authorized roles here. Use 'all' to authorize anyone
+})
+```
+
+and we will project the whole entity
+
+```typescript
+@Projects(Post, "id")
+  public static projectPost(entity: Post, currentPostReadModel?: PostReadModel): PostReadModel {
+    return new PostReadModel(entity.id, entity.title, entity.content, entity.author)
+  }
+```
+
+## 6. Deployment
+Everything we need for a basic project is set. It is time to deploy it:
+
+`boost deploy -e production`
+
+It will take a couple of minutes to deploy all the resources.
+
+When the the serverless backend is successfully deployed you will see information about your stack endpoints. For this basic project we will only need to pick the REST API endpoint, reflected in the output as `backend-application-stack.baseRESTURL`, e.g.:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/`
+
+We will use this endpoint to test our backend.
+
+## 7. Testing
+Let's get started testing the project. We will perform two actions:
+- Add a couple of posts
+- Retrieve all posts
+- Retrieve a specific post
+
+To perform the HTTP requests, you might want to use something like `Postman`, although `curl` would also work.
+
+### 7.1 Creating posts
+We will perform a two `POST` request to the following commands URL:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/commands`
+
+The first request:
+
+```json
+{
+	"typeName": "CreatePost",
+	"version": 1,
+	"value": {
+    	"postId": "95ddb544-4a60-439f-a0e4-c57e806f2f6e",
+    	"title": "This is my first post",
+    	"content": "I am so excited to write my first post",
+		"author": "Some developer"
+  }
+}
+```
+
+The second request:
+
+```json
+{
+	"typeName": "CreatePost",
+	"version": 1,
+	"value": {
+    	"postId": "05670e55-fd31-490e-b585-3a0096db0412",
+    	"title": "This is my second post",
+    	"content": "I am so excited to write my second post",
+		"author": "Some developer"
+  }
+}
+```
+
+We should now have two `Posts` in our backend, no authorization header is required since we have allowed `all` access to our commands and read models.
+
+### 7.2 Retrieving all posts
+In order to retrieve the information we just sent, lets perform a `GET` request that will be hitting our read model `PostReadModel`:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/readmodels/PostReadModel`
+
+### 7.3 Retrieving specific post
+It is also possible to retrieve specific a `Post` by adding the `postId` to the URL, e.g.:
+
+`https://<api-gateway-id>.execute-api.us-east-1.amazonaws.com/production/readmodels/PostReadModel/95ddb544-4a60-439f-a0e4-c57e806f2f6e`
+
```

Comment:
Can you rewrite this section by using GraphQL?

Reply:
what I'm wondering is whether we want to get people straight into an extra layer of complexity with GraphQL 🤔

Reply:
Rewrote everything using graphql, thanks @NickSeagull and @alvaroloes